tests/layer_tests/requirements.txt

```diff
@@ -12,6 +12,10 @@ transformers
 packaging
 pillow
 pytest
+networkx
+defusedxml
+fastjsonschema
```

Comment:
looks not used

Reply:
let us try to remove it in the next PR

Reply:
defusedxml is use , not sure about others